crm_stage_type/tests/test_crm_stage_type.py

```diff
@@ -16,26 +16,54 @@ def setUp(self):
             'groups_id': [(6, 0, [self.ref('sales_team.group_sale_manager')])]
         })
```

Comment:
as you are improving tests, would you mind replacing TransactionCase with Savepoint case and replace setUp like this?
```py
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True, no_reset_password=True))
        cls.crm_salemanager = cls.env['res.users'].create({
            'company_id': cls.env.ref("base.main_company").id,
            'name': "Crm Sales manager",
            'login': "csm",
            'email': "crmmanager@yourcompany.com",
            'groups_id': [(6, 0, [cls.ref('sales_team.group_sale_manager')])]
        })

Reply:
Changed it to SavepointCase